client/auth/controller.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import ReactDom from 'react-dom';
```

Comment:
>In the future we could even shorten it to:

Worth discussing, sure. Sometimes I worry that the brevity of it loses important context, i.e. in this case, not as clear that it's using React's render and not some local helper function I've declared.